lang/cs/Org.Apache.REEF.Examples.AllHandlers/Org.Apache.REEF.Examples.AllHandlers.csproj

```diff
@@ -33,6 +33,7 @@ under the License.
     <StartupObject />
   </PropertyGroup>
   <Import Project="$(SolutionDir)\build.props" />
+  <Import Project="$(SolutionDir)\build.targets" />
```

Comment:
Why do we need to import build.targets in this project? I've just tried clean build without this line, and it succeeded as well.